hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -3285,6 +3295,49 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="bitsTimestamp" default="41" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets the timestamp component bit length, default 41 bits.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:simpleType>
+                <xs:restriction base="xs:int">
+                    <xs:minInclusive value="1" />
```

Comment:
`maxInclusive` can be set to 63 (if seuqence bits is 0 and node ID bits is 1)

Reply:
done.